zetasql/parser/bison_parser.y

```diff
@@ -3435,11 +3460,102 @@ privilege_name:
       {
         $$ = $1;
       }
-    | KW_SELECT
+    | "SELECT"
       {
         // The SELECT keyword is allowed to be a privilege name.
         $$ = parser->MakeIdentifier(@1, parser->GetInputText(@1));
       }
+    | "CREATE"
+      {
+        $$ = parser->MakeIdentifier(@1, parser->GetInputText(@1));
+      }
+    | "INDEX"
+      {
+        $$ = parser->MakeIdentifier(@1, parser->GetInputText(@1));
+      }
+    | "ALTER" "USER"
```

Comment:
just concern about extensibility. 

Those two word privileges do require parser update for every new privilege, or it better to replace e.g 'CREATE USER' to 'CREATEUSER'. 

Anyway you can reserve some common parts of the two words privileges, as synonyms. MySQL dialect is ugly  :(

Reply:
combine the two words looks more strange. how to set as synonyms

Reply:
not parser side, but privilege impl. Just point it out here